kubernetes/AKS/README.md

```diff
@@ -118,11 +118,11 @@ nvidia-device-plugin-daemonset-qccgn   1/1     Running   0          144m
 
 ```bash
 wget https://torchserve.pytorch.org/mar_files/squeezenet1_1.mar
-wget https://torchserve.pytorch.org/mar_files/mnist.mar
+wget https://torchserve.pytorch.org/mar_files/mnist_v2.mar
```

Comment:
@harshbafna The https://torchserve.pytorch.org/mar_files/mnist.mar is currently pointing to the new mar file generated by @abishekchiffon 
`mnist_v2` doesn't contain any envelope changes.

Reply:
@maaquib: Thanks for the update.

@HamidShojanazeri: Could you validate the https://torchserve.pytorch.org/mar_files/mnist.mar with updated docker image built from source?

If that works fine I will revert these changes and make the changes in the model zoo with the new URL.